modules/kubernetes/pages/k8s-operator/backup-and-restore.adoc

```diff
@@ -0,0 +1,710 @@
+  If destination is s3:
+    --s3-bucket :               S3 Bucket name
+    --aws-access-key-id :       the aws access key id
+    --aws-secret-access-key :   the aws secret access key
+----
+
+== Create a backup
+
+=== Back up to local storage
+
+Use the following command to back up a cluster and store backup files in local storage.
+In this example, the cluster is called `test-cluster`.
+
+[source.wrap, console]
+----
+ kubectl tg backup create --cluster-name test-cluster --tag testlocal -n tigergraph
+  --destination local \
+  --local-path /home/tigergraph/backup
+----
+
+You can also customize the timeout, staging path, and the compress process number.
+
+[source.wrap, console]
+----
+ kubectl tg backup create --cluster-name test-cluster --tag testlocal -n tigergraph
+  --destination local \
+  --local-path /home/tigergraph/backup  --staging-path /home/tigergraph/temp \
+  --timeout 18000 --compress-process-number 0
+----
+
+=== Back up to S3 bucket
+
+Use the following command to back up a cluster and store the backup files in S3.
+In this example, the cluster is called `test-cluster`.
+
+You will need the S3 bucket name, access key id and S3 secret.
+
+[source.wrap, console]
+----
+kubectl tg backup create --cluster-name test-cluster --tag testS3 -n tigergraph
+--destination s3Bucket \
+--s3-bucket tgbackup --aws-access-key-id $ACCESS_KEY_ID \
+--aws-secret-access-key $SECRET_ACCESS_KEY
+----
+
+You can also customize the timeout, staging path, and the compress process number.
+
+[source.wrap, console]
+----
+kubectl tg backup create --cluster-name test-cluster --tag testS3 -n tigergraph
+--destination s3Bucket \
+--s3-bucket tgbackup --aws-access-key-id $ACCESS_KEY_ID \
+--aws-secret-access-key $SECRET_ACCESS_KEY \
+--staging-path /home/tigergraph/temp \
+--timeout 18000 --compress-process-number 0
+----
+
+== Incremental backup
+
+=== Update an existing backup
+
+If you have already created one backup with `kubectl tg backup create` , you can use `kubectl tg backup update` to change the configurations of backup. Once you run the update command, the backup will start immediately with the updated configurations.
+
+[NOTE]
+====
+A cluster and a tag correspond to a backup configuration. For example:
+
+* `--cluster-name test-cluster --tag tests3` corresponds to a CR named `test-cluster-backup-tests3`.
+* `--cluster-name test-cluster --tag testlocal` corresponds to a CR named `test-cluster-backup-testlocal`.
+
+These two have the same target cluster but different tags, so they correspond to different CRs.
+====
+
+You have already created a backup using `kubectl tg backup create`:
+
+[source.wrap, console]
+----
+kubectl tg backup create --cluster-name test-cluster --tag testlocal -n tigergraph --destination local --local-path /home/tigergraph/backup  --staging-path /home/tigergraph/temp  --timeout 18000 --compress-process-number 0
+----
+
+To change the timeout, run:
```

Comment:
Colons should follow complete sentences or at least complete clauses.  
(I make a statement) : ( an example that supports my statement)

They should not be used to separate a verb phrase (e.g. "run", "create") from an example that acts as the object of the verb phrase. Because we don't put separates between verbs and their objects.